### ISSUE TITLE
[BUG] KeyError about 'state_value' from PPOLoss

### ISSUE THREAD

**Issue by xmaples** [pytorch/rl#1097]
## Describe the bug

A KeyError will be raised from `PPOLoss` when PPO working with `ActorCriticOperator`

This is caught by tensor getting using a hardcoded key name 'state_value', on code line: https://github.com/pytorch/rl/blob/v0.1.0/torchrl/objectives/ppo.py#L127  .
However, `ActorCriticOperator` produces the tensor naming 'state_action_value'.

This may be fixed using a value_key parameter instead of hardcoded key name.

## To Reproduce

Steps:
 1. create `ActorCriticOperator` and PPOLoss (e.g. ClipPPOLoss)
 2. run loss

```python
# reproduce KeyError raised from ppo loss with ActorCriticOperator
""" versions of libraries
torchrl 0.1.0
gym 0.26
torch 2.0
mujoco
"""

from torchrl.envs.libs.gym import GymEnv
from torchrl.envs.transforms import TransformedEnv,DoubleToFloat
from torchrl.modules import ActorCriticOperator, ProbabilisticActor, ValueOperator, Actor
from torchrl.modules import MLP
from torchrl.modules.distributions import TanhNormal
from torchrl.objectives.value import GAE
from torchrl.objectives.ppo import ClipPPOLoss
from torchrl.collectors import SyncDataCollector
from tensordict.nn import TensorDictModule
from tensordict.nn.distributions import NormalParamExtractor
import torch
from torch import nn

env = GymEnv("InvertedDoublePendulum-v4")
print(env.action_spec)
env=TransformedEnv(env, DoubleToFloat(in_keys=["observation"]))

ac_common_module = TensorDictModule(
    module=nn.LazyLinear(out_features=32),
    in_keys=["observation"],
    out_keys=["hidden"]
)
policy_module = ProbabilisticActor(
    module=TensorDictModule(
        module=nn.Sequential(
            nn.LazyLinear(out_features=2*env.action_spec.shape[-1]),
            NormalParamExtractor()
        ),
        in_keys=["hidden"],
        out_keys=["loc","scale"]
    ),
    in_keys=["loc", "scale"],
    distribution_class=TanhNormal,
    distribution_kwargs={
        "min": env.action_spec.space.minimum,
        "max": env.action_spec.space.maximum,
    },
    return_log_prob=True,
)
critic_module = ValueOperator(
    module=MLP(out_features=1, depth=0),
    in_keys=["hidden", "action"],
    out_keys=["state_action_value"]  # same as the default key name
)
ac_op = ActorCriticOperator(ac_common_module, policy_module, critic_module)
policy_op = ac_op.get_policy_operator()
critic_op = ac_op.get_critic_operator()

ac_op(env.reset())

adv_module = GAE(
    gamma=0.99,
    lmbda=0.95,
    value_network=critic_op,
    value_key="state_action_value"
)

loss_module = ClipPPOLoss(
    actor=policy_op,
    critic=critic_op
)

collector = SyncDataCollector(
    env,
    policy=policy_op,
    frames_per_batch=4,
    total_frames=16)

for td in collector:
    with torch.no_grad():
        adv_module(td)
    loss_module(td)
    break


```

## Screenshots

![KeyError](https://user-images.githubusercontent.com/5900204/234757693-d7060301-cca2-464f-84cb-eac8dec28b59.png)

## Reason and Possible fixes

See in Description.
